src/components/datePicker/DatePickerDropdown.tsx

```diff
@@ -157,9 +157,13 @@ export class DatePickerDropdown extends React.Component<IDatePickerDropdownProps
         </footer>
       )
     };
-    let datePickerBox: JSX.Element = this.props.withReduxState
-      ? <DatePickerBox withReduxState id={this.props.id} {...datePickerBoxProps} />
-      : <DatePickerBox {...datePickerBoxProps} />;
+
+    let datePickerBox: JSX.Element = null;
```

Comment:
Have you tested thorougly that initializing this to null doesn't break elsewhere, especially since before it was always instantiated? Kind of nifty stuff